src/utils.js

```diff
@@ -0,0 +1,38 @@
+export const cleanMovies = (movies) => {
+    return movies.map(movie => {
+        return {
+            id: movie.id,
+            title: movie.title,
+            poster_path: movie.poster_path
+        }
+    })
+}
+
+export const cleanMovie = (selectedMovie) => {
```

Comment:
You could consider destructuring right in the parameter here to avoid more lines of code?